lib/main.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_interns/assesment_1/fi_ahmad.dart';
```

Comment:
I believe this import statement is not required anymore as we are not using FIAhmad(), please take a look